From 447f4ece97d153efff8ab8ab398e308043b5c67f Mon Sep 17 00:00:00 2001
From: John Fitzpatrick <john@cyberfitz.org>
Date: Tue, 12 Aug 2025 17:30:31 -0700
Subject: [PATCH 17/38] Added .claude/agents to .gitignore

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index fbeaa6d..c393f55 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,4 @@ __pycache__
 .serena
 .claude/settings.local.json
 PRPs/local
+.claude/agents
\ No newline at end of file
-- 
2.39.5

